astropy/wcs/wcsapi/tests/test_fitswcs.py

```diff
@@ -1591,7 +1591,7 @@ def test_array_index_conversions_arrays_1d():
     wcs = WCS(naxis=1)
     wcs.wcs.ctype = ("FREQ",)
 
-    coord = SpectralCoord([10, 12], unit="GHz")
+    coord = SpectralCoord([10, 12], unit="Hz")
```

Comment:
I don't understand why this change is needed for armv7. Can you please clarify? Thanks!

Reply:
Because with GHz, the pixel coordinate value ended up being very big, and it triggered a numerical warning, so I've made it so the values of the world coordinates are closer to the crval of the WCS. Not sure why arm specifically saw the warning, but not surprised to see numerical warnings on some platforms and not others.